lib/crowbar/init/application.rb

```diff
@@ -142,6 +142,17 @@ def cleanup_db
           end
         end
 
+        def migrate_db_rake
```

Comment:
Why do we migrate the database? This is automatically done while starting crowbar.